namespace-provisioner/provision-developer-ns.md

```diff
@@ -131,4 +131,11 @@ by Namespace Provisioner.
   limitrange/dev-lr   2023-03-08T04:22:20Z
   ```
 
-  For more information, see the GitOps section of [Customize Installation of Namespace Provisioner](customize-installation.md).
+## <a id ='additional-users-k8s-rbac'></a>Enable additional users with Kubernetes RBAC
+  
+Namespace Provisioner does not yet support enabling additional users with Kubernetes RBAC (support will be provided in an upcoming release).
+Until Namespace Provisioner support is provided, please follow the instructions [here](legacy-manual-namespace-setup.hbs.md#additional-user-access).
+
+----
```

Comment:
This is not correct. The following line should be inside the Gitops tab
```
For more information, see the GitOps section of [Customize Installation of Namespace Provisioner](customize-installation.md).
```
and the new section Enable additional users with Kubernetes RBAC should start after that

Reply:
Thanks. I've made that update in [8b5c808](https://github.com/pivotal/docs-tap/pull/2461/commits/8b5c8081a78d82e54a82957b90a724f001596fac)